rllib/algorithms/impala/impala.py

```diff
@@ -219,8 +217,8 @@ def training(
             timeout_s_aggregator_manager: The timeout for waiting for replay worker
                 results -- typically if this is too low, the manager won't be able to
                 retrieve ready replay requests.
-            broadcast_interval: Max number of workers to broadcast one set of
-                weights to.
+            broadcast_interval: Number of training step calls before weights are
```

Comment:
Thanks for fixing this obviously wrong description :)

Actually, I don't think evaluation workers have anything to do with this here.
Also, don't we only update those workers that had sent samples? Or did we change this logic?

Reply:
both are good points. thanks.